source/module.coffee

```diff
@@ -165,11 +170,47 @@ class Space.Module extends Space.Object
     this[hook] ?= ->
     this[hook] = _.wrap(this[hook], wrapper)
 
+  _setupLogger: ->
+    config = Space.configuration.log
```

Comment:
So we can do away with the `Space.configuration` as that was only needed since we were previously initialising the logger instance outside the module chain. This will be really nice to see gone